cli/lsp/tsc.rs

```diff
@@ -499,8 +499,18 @@ impl NavigationTree {
     specifier: &ModuleSpecifier,
     source: &CodeLensSource,
   ) -> lsp::CodeLens {
+    let range = if let Some(name_span) = &self.name_span {
```

Comment:
ran into a bug when typing in the editor where sometimes the `name_span` can be undefined, so hardened this whole area